.travis.yml

```diff
@@ -141,6 +141,7 @@ branches:
     # bors-ng branches; see https://bors-ng.github.io/getting-started/
     - trying
     - staging
+    - issue-947
```

Comment:
This line shouldn't be necessary.  Travis automatically builds PRs.  This line is only necessary if you want nix-rust/nix to have a branch named "issue-947".